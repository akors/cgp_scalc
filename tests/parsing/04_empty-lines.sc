1 + 1


2+1+4+5


99 * 21
