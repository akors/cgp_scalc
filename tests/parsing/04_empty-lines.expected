2
12
2079
